matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/sync/SyncResponseHandler.kt

```diff
@@ -101,8 +103,8 @@ internal class SyncResponseHandler @Inject constructor(
         }
     }
 
-    private fun startCryptoService(isInitialSync: Boolean) {
-        relevantPlugins.measureSpan("task", "start_crypto_service") {
+    private fun List<SpannableMetricPlugin>.startCryptoService(isInitialSync: Boolean) {
```

Comment:
nitpick: I'm not sure it's right for `startCryptoService` to be a an extension on `List<SpannableMetricPlugin>` because it should be possible to call `startCryptoService` independent of the measurement IMO.

Reply:
This is a private fun of `SyncResponseHandler`, so I think this is acceptable.
Using extensions of `List<SpannableMetricPlugin>` is not ideal, but the change in the code was minimal doing like this.